app/middleware/error_handler.py

```
@@ -0,0 +1,512 @@
+from typing import Any, Callable, Dict, Optional
+from uuid import UUID
+
+from fastapi import FastAPI, Request, Response, status
+from fastapi.exceptions import RequestValidationError, HTTPException
+from fastapi.responses import JSONResponse
+from pydantic import ValidationError as PydanticValidationError
+from starlette.exceptions import HTTPException as StarletteHTTPException
+
+from app.core.exceptions import (
+    MobiusException,
+    ValidationError,
```

Ruff (0.11.9)

`app.core.exceptions.ValidationError` imported but unused

Remove unused import: `app.core.exceptions.ValidationError`

(F401)